blueprints/ember-frost-object-browser/index.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-core', target: '>=0.0.14 <2.0.0'},
-        {name: 'ember-frost-bunsen', target: '^3.0.0'},
+        {name: 'ember-frost-core', target: '>=0.1.0 <2.0.0'},
```

Comment:
hmm, I wonder why that was? It doesn't look like it should have:
https://github.com/ciena-frost/ember-frost-bunsen/blob/master/blueprints/ember-frost-bunsen/index.js#L13